packages/rocketchat-lib/server/methods/sendMessage.coffee

```diff
@@ -49,8 +49,9 @@ Meteor.methods
 		message.alias = user.name if not message.alias? and RocketChat.settings.get 'Message_SetNameToAliasEnabled'
 		if Meteor.settings.public.sandstorm
 			message.sandstormSessionId = this.connection.sandstormSessionId()
-
+		
 		RocketChat.sendMessage user, message, room
+		RocketChat.metrics.messagesSent.inc()
```

Comment:
I propose we remove the sendMessage hook here and work on a PR to make use of the statistics collection we are already creating.  That way we don't adversely effect things